### PR TITLE
Add Dynamic Set Dollar (DSD) as collateral

### PR DESCRIPTION
**Pull request by akshaysrivastav**

### CODE REVIEW

File: UMIPs/umip-draft_DSD_collateral.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+| Authors    | John (dsdravage@gmail.com), Sean (sean@opendao.io) |
+| Status     | Draft                                                                                                                                    |
+| Created    | January 13, 2021                                                                                                                           |
+ 
+## Summary (2-5 sentences)
+This UMIP will add DSD to the supported collateral currencies into the global whitelist contract, allowing the usage of DSD as collateral currency.
+
+​
+## Motivation
+Dynamic Set Dollar (DSD) is a fast growing algorithmic stablecoin. DSD has a current market capitalization of $128 million & $28.7 million in liquidity. The volume since inception (under two months) has exceeded $444 million at the time of this writing.
```
Comment (Tommy1231232) [inline UMIPs/umip-draft_DSD_collateral.md:14]: Really excited for this!!

File: UMIPs/umip-draft_DSD_collateral.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+The motivation of adding DSD as a supported collateral in the global whitelist contract is so that DSD holders can go long on DSD by minting a stablecoin backed by DSD as collateral, which will be built on UMA.
+
+DSD holders would supply tokens to the OpenDAO Platform as a collateral asset to mint DSDO via underlying UMA contracts and use the minted DSDO elsewhere. DSDO being a synthetic asset that traces the value of USD. Once users of the DSDO token want their locked DSD collateral back they would bring DSDO back to the OpenDAO platform, settle any fees/interest and reclaim DSD tokens. A DSDO/USDC pool will be created via uniswap & sushiswap for easy access into other markets using DSDO.
+
+​
+​
+## Technical Specification
+To accomplish this upgrade, two changes need to be made:
+- The DSD address, [0xBD2F0Cd039E0BFcf88901C98c0bFAc5ab27566e3](https://etherscan.io/token/0xBD2F0Cd039E0BFcf88901C98c0bFAc5ab27566e3), needs to be added to the collateral currency whitelist introduced in UMIP-8. 
+- A final fee of 571 DSD needs to be added for the DSD in the Store contract.
```
Comment (smb2796) [inline UMIPs/umip-draft_DSD_collateral.md:25]: Why was 571 chosen? Seems like an extremely specific number

Comment (akshaysrivastav) [reply UMIPs/umip-draft_DSD_collateral.md:25]: Hey @smb2796, we were told by UMA team that the final fee value should be close to $400, so we chose 571. Please let us know if this needs to be changed (a rounded off number can be used like 550 or 600).

Comment (chrismaree) [reply UMIPs/umip-draft_DSD_collateral.md:25]: ye, I think a round number is nicer. something like 600 would be ideal. Ideally, the final fees should be ~400 so 600 would be good.

File: UMIPs/umip-draft_DSD_collateral.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+​
+​
+## Rationale
```
Comment (Tommy1231232) [inline UMIPs/umip-draft_DSD_collateral.md:29]: Can you explain the mechanism by which DSD maintains its peg to $1? We want to make sure our EMP will be able to support this collateral currency. Today, we cannot support rebasing or a-style tokens (where the token balance changes)

**Comment by Tommy1231232** [comment]
If the minor comments above are addressed we can get it up for a vote this week!

File: UMIPs/umip-draft_DSD_collateral.md (hunk 2 above)
Comment (akshaysrivastav) [reply UMIPs/umip-draft_DSD_collateral.md]: Thanks guys, updated the fee value to 600.

File: UMIPs/umip-draft_DSD_collateral.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+## Motivation
+Dynamic Set Dollar (DSD) is a fast growing algorithmic stablecoin. DSD has a current market capitalization of $128 million & $28.7 million in liquidity. The volume since inception (under two months) has exceeded $444 million at the time of this writing. 
+
+The motivation of adding DSD as a supported collateral in the global whitelist contract is so that DSD holders can go long on DSD by minting a stablecoin backed by DSD as collateral, which will be built on UMA.
+
+DSD holders would supply tokens to the OpenDAO Platform as a collateral asset to mint DSDO via underlying UMA contracts and use the minted DSDO elsewhere. DSDO being a synthetic asset that traces the value of USD. Once users of the DSDO token want their locked DSD collateral back they would bring DSDO back to the OpenDAO platform, settle any fees/interest and reclaim DSD tokens. A DSDO/USDC pool will be created via uniswap & sushiswap for easy access into other markets using DSDO.
+
+​
+​
+## Technical Specification
```
Comment (chrismaree) [inline UMIPs/umip-draft_DSD_collateral.md:22]: This is great. can you please add a few sentences or bullet points to make it clear that DSD is _not_ like other algorithmic stable coins wherein the supply can dynamically expand/contract within re-bases? This is important as this kind of dynamic supply token is not compatible with the DVM or downstream UMA financial contracts.

Comment (akshaysrivastav) [reply UMIPs/umip-draft_DSD_collateral.md:22]: Just added a note.

**Review by Tommy1231232 — APPROVED** [review]

**Pull request merged**